src/main.cpp

```diff
@@ -19,6 +19,17 @@
 #include "ShaderTool.h"
 #include "InputManager.h"
 
+InputManager* inputManager = nullptr;
+
+void scroll_callback(GLFWwindow* window, double xoffset, double yoffset)
+{
+		if(inputManager == NULL){
+			std::cout << "WHOOPS: inputManager is NULL\n";
+			return;
+		}else{
```

Comment:
Left bracket needs to go to next line to keep consistent with the rest of the code.

else should go to new line as well.

Reply:
d